test/module/child_process/basics.js

```diff
@@ -0,0 +1,30 @@
+// Test basic child process functionality
+//
+
+async_test(function(){
+    var text = "hello";
+    var process = require('child_process');
+    var p = process.spawn("echo", ["-n", text]);
```

Comment:
****cough**** Seems like our life would be so much easier with Chromium + Node.js. :wink: